homeassistant/components/enasolar/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "enasolar",
+  "name": "EnaSolar Solar Inverter",
+  "documentation": "https://www.home-assistant.io/integrations/enasolar",
+  "requirements": [
+      "pyenasolar==0.1.4"
+  ],
+  "codeowners": ["@geustace"],
+  "iot_class": "local_polling",
```

Comment:
Is the device discoverable by any of the built-in discovery methods?
https://developers.home-assistant.io/docs/creating_integration_manifest#zeroconf

Reply:
Not that I am aware of. It has a web interface, so I assume that it might work with zeroconf, but I would prefer to leave that to a future version